apps/entities/api/serializers.py

```diff
@@ -1,27 +1,21 @@
-"""Serializers for the entities app."""
+from rest_framework import serializers
 
-from typing import TYPE_CHECKING
+from apps.entities.models.entity import Entity
+from core.models.module import TypedModuleSerializerDrf
```

Comment:
nit: Can we do `DrfBlahBlahSerializer` or `BlahBlahDrfSerializer` instead of `BlabBlahSerializerDrf`?